wandb/apis/public.py

```diff
@@ -188,6 +188,7 @@ class RetryingClient:
         query ServerInfo{
             serverInfo {
                 cliVersionInfo
+                licenseFlags
```

Comment:
i saw that this one was added 5 month ago. Do we care to check servers that are older than this?

Reply:
good call, I'll think on this a bit more.